kudo-operator/templates/spark-events-pvc.yaml

```diff
@@ -0,0 +1,14 @@
+{{ if .Params.enableHistoryServer }}
```

Comment:
based on the previous comment, let's add `createPersistentVolumeClaim` to the condition